model/remote_cluster.go

```diff
@@ -149,10 +149,6 @@ func (m *RemoteClusterMsg) IsValid() *AppError {
 		return NewAppError("RemoteClusterMsg.IsValid", "api.remote_cluster.invalid_id.app_error", nil, "Id="+m.Id, http.StatusBadRequest)
 	}
 
-	if m.Topic == "" {
-		return NewAppError("RemoteClusterMsg.IsValid", "api.remote_cluster.invalid_topic.app_error", nil, "Topic empty", http.StatusBadRequest)
-	}
-
```

Comment:
I've temporarily removed this since it was causing failures. We actually send and receive some messages without topic (ping and invitation).
Let me know how you'd like to handle these cases.

Reply:
I have a PR coming up that includes a topic for pings and invitations.  I'll add this check back when that gets submitted.